src/dparse/ast.d

```diff
@@ -894,6 +894,7 @@ final class AtAttribute : BaseNode
     /** */ ArgumentList argumentList;
     /** */ TemplateInstance templateInstance;
     /** */ Token identifier;
+    /** */ bool useParen;
```

Comment:
this could probably easily be made into a property function checking `this.tokens` to save some memory (and alignment bytes)